task2/task2.2.js

```diff
@@ -0,0 +1,175 @@
+const { v4: get_uuid} = require('uuid');

```

Comment:
the result of both tasks is only one application. You will improve this in next tasks